modules/nextMillenniumBidAdapter.js

```diff
@@ -200,6 +241,32 @@ function getTopWindow(curWindow, nesting = 0) {
   }
 }
 
+function getAd(bid) {
+  let ad, adUrl, vastXml, vastUrl
+
+  switch (deepAccess(bid, 'ext.prebid.type')) {
+    case VIDEO:
```

Comment:
The response is being parsed for video however, I don't see a video request being generated in `buildRequests()`. Which video formats are you looking to support (outstream or instream)? 

Ref: https://docs.prebid.org/prebid-video/video-getting-started.html